widgets/dialogs.pas

```diff
@@ -278,6 +287,33 @@ procedure TMessageDialog.KeyDown(var Key: NativeInt; Shift: TShiftState);
   end;
 end;
 
+procedure TMessageDialog.Show;
+var
+  ownForm: TWForm;
+  curHeight: Integer;
+begin
+  inherited Show;
+  if not (Owner is TWForm) then
+    exit;
```

Comment:
`Exit`, `Break` and `Continue` should be written as Identifiers in PascalCase (cause in fact they _are_ identifiers, one can overload them with e.g. variables 😋)

Reply:
Fixed.